packages/webapp/src/Routes.jsx

```diff
@@ -553,8 +553,8 @@ const Routes = () => {
             <Route path="/fence/:location_id" component={FenceDetails} />
             <Route path="/buffer_zone/:location_id" component={BufferZoneDetails} />
             <Route path="/watercourse/:location_id" component={WatercourseDetails} />
-            <Route path="/sensor/:location_id" component={SensorDetails} />
-
+            <Route path="/sensor/:location_id/details" exact component={SensorDetail} />
+            <Route path="/sensor/:location_id/edit" exact component={EditSensor} />
```

Comment:
The route should still be `<Route path="/sensor/:location_id" component={SensorDetails} />`. SensorDetails is a collection of routes that holds the rest of the sensor view routes. Similar to WatercourseDetails